samples/typetest/typetest.cpp

```diff
@@ -34,11 +34,7 @@
 
 #include "wx/ioswrap.h"
```

Comment:
Shouldn't this one be removed or replaced, especially in the samples, now that it's obsolete?
It's also present in `samples/docview/doc.cpp` and `samples/widgets/textctrl.cpp`.

Reply:
Good point, thanks!